src/libraries/Microsoft.PowerFx.Interpreter/Functions/Library.cs

```diff
@@ -1043,6 +1065,17 @@ static Library()
                     returnBehavior: ReturnBehavior.AlwaysEvaluateAndReturnResult,
                     targetFunction: Replace)
             },
+            {
+                BuiltinFunctionsCore.RGBA,
+                StandardErrorHandling<NumberValue>(
+                    BuiltinFunctionsCore.RGBA.Name,
+                    expandArguments: NoArgExpansion,
+                    replaceBlankValues: ReplaceBlankWithZero,
+                    checkRuntimeTypes: ExactValueTypeOrBlank<NumberValue>,
+                    checkRuntimeValues: DeferRuntimeValueChecking,
+                    returnBehavior: ReturnBehavior.ReturnBlankIfAnyArgIsBlank,
```

Comment:
>ReturnBlankIfAnyArgIsBlank [](http://example.com/codeflow?start=51&length=26)

AlwaysEvaluateAndReturnResult. RGBA(Blank(), Blank(), 255, 1) should be the same as RGBA(0,0,255,1)

Reply:
Fixed and added tests